CHANGELOG

```diff
@@ -1,6 +1,17 @@
 devel
 -----
 
+* APM-173: Now, arangobench supports multiple coordinators, so the flag 
+  `--server.endpoint` can be used multiple times, as in the example below:
+
+  arangobench \
+    --server.endpoint tcp://[::1]::8529 \
+    --server.endpoint tcp://[::1]::8530 \
+    --server.endpoint tcp://[::1]::8531 
+
+  This does not compromise the use of the other client tools, which preserve
+  the behavior of having one coordinator.
```

Comment:
What about arangodump and arangorestore? They are going to be changed by this PR, too...